network-api/networkapi/donate/factory.py

```diff
@@ -18,6 +18,13 @@ class Meta:
     featured_image = SubFactory(ImageFactory)
 
 
+class DonateHelpPageFactory(PageFactory):
+    class Meta:
+        model = DonateHelpPage
+
+    title = Faker("text", max_nb_chars=140)
```

Comment:
Just FYI, I'm usually using `Faker("sentence")`, since it's usually better for a phrase.